projects/natural/src/lib/modules/columns-picker/columns-picker.component.ts

```diff
@@ -28,19 +28,35 @@ export class NaturalColumnsPickerComponent implements AfterViewInit, OnDestroy {
     }
 
     /**
-     * Emit a list of column keys whenever the selection changes
+     * Define preselected (checked) columns at start
      */
-    @Output() public readonly selectionChange = new EventEmitter<string[]>();
-    @Output() public readonly defaultSelectionChange = new EventEmitter<string[]>();
+    private _selections?: string[];
+    @Input()
```

Comment:
```suggestion
    /**
     * Set the columns that are wanted but might be unavailable.
     *
     * If a column is unavailable it will be ignored silently. To know what columns were actually applied
     * you should use `selectionChange`.
     * 
     * It is often set once on component initialization, but it can also be set again later in the lifespan of the component.
     */
    @Input()
```